### DISCUSSION TITLE
ETA for next release?

### DISCUSSION THREAD

**Discussion by almightykid** [strimzi/drain-cleaner#131] [Q&A]
Hello there, 

When is it planned to generate a new release for the helm chart including the latest changes? 

Thanks in advance!

**Comment by scholzj** [comment]
@almightykid I'm away this week. But I will have a look at it later this week or next week. I'm sure we can do a new patch release if there is interest.

**Reply by almightykid** [reply]
Hey @scholzj thanks for the fast reply. That would be lovely, as we would like to use the annotations block that I submitted some time ago. Whenever you can do it is fine :)

**Reply by scholzj** [reply]
FYI: I created the RC1. So unless any problems are found, the final release should be out sometime next week.

**Reply by almightykid** [reply]
@scholzj thank you very much :)

**Reply by scholzj** [reply]
Ok, the GA should be out now!
